packages/php-storage-driver-common/contract/Driver/Command/DriverCommandHandlerInterface.php

```diff
@@ -5,13 +5,15 @@
 namespace Keboola\StorageDriver\Contract\Driver\Command;
 
 use Google\Protobuf\Internal\Message;
+use Keboola\StorageDriver\Command\Common\DriverResponse;
 use Keboola\StorageDriver\Command\Common\RuntimeOptions;
 
 interface DriverCommandHandlerInterface
 {
     /**
      * @param string[] $features
      * @param RuntimeOptions $runtimeOptions
+     * @return DriverResponse|Message|null
```

Comment:
Přemýšlal sem že udělám BC a dám to přímo do parametrů, ale pak sem si to rozmyslel.